exp/services/ledgerexporter/internal/queue_test.go

```diff
@@ -56,7 +56,7 @@ func TestQueue(t *testing.T) {
 	require.Equal(t, float64(0), getMetricValue(queue.queueLengthMetric).GetGauge().GetValue())
 	require.Equal(t, uint32(3), l.GetStartLedgerSequence())
 
-	l, ok, err = queue.Dequeue(context.Background())
+	l, _, err = queue.Dequeue(context.Background())
 	require.NoError(t, err)
 	require.False(t, false)
```

Comment:
```suggestion
	l, ok, err = queue.Dequeue(context.Background())
	require.NoError(t, err)
	require.False(t, ok)
```

looks like a prexisting bug